### PR TITLE
Load scene

### PR DESCRIPTION
**Pull request by wouterz**
Added a few objects to resemble the cockpit

### CODE REVIEW

**Comment by dalderliesten** [comment]
Looks nice!

File: src/main/java/nl/tudelft/pixelperfect/Game.java [hunk 1]
```diff
@@ -3,10 +3,16 @@
...
 import com.jme3.network.Network;
 import com.jme3.network.Server;
 import com.jme3.network.serializing.Serializer;
 
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Spatial;
+import com.jme3.scene.shape.Box;
 import nl.tudelft.pixelperfect.client.ConnectListener;
 import nl.tudelft.pixelperfect.client.HelloMessage;
 import nl.tudelft.pixelperfect.client.ServerListener;
```
Comment (FlorisXll) [inline src/main/java/nl/tudelft/pixelperfect/Game.java]: I have some Checkstyle warnings here: the imports are in the wrong order (local packages come first).

File: src/main/java/nl/tudelft/pixelperfect/Game.java [hunk 2]
```diff
@@ -59,6 +69,56 @@ public void simpleInitApp() {
...
+    Box floor = new Box(4, 0.01f, 4);
+    Geometry g2 = new Geometry("Floor", floor);
+    Material mat2 = new Material(assetManager, "Common/MatDefs/Misc/Unshaded.j3md");
+    mat2.setColor("Color", ColorRGBA.Green);
+    g2.setMaterial(mat2);
+    g2.setLocalTranslation(0, -1, 0);
+    rootNode.attachChild(g2);
+
+    //walls
+    Box wall_left = new Box(0.01f, 4, 4);
```
Comment (FlorisXll) [inline src/main/java/nl/tudelft/pixelperfect/Game.java]: Checkstyle warning: wall_left --> wallLeft.

File: src/main/java/nl/tudelft/pixelperfect/Game.java [hunk 3]
```diff
@@ -120,6 +125,21 @@ public void createMap() {
...
+   */
+  public void drawTimer() {
+    guiNode.detachAllChildren();
+    guiFont = assetManager.loadFont("Interface/Fonts/Default.fnt");
+    BitmapText timer = new BitmapText(guiFont, false);
+//    timer.setSize(1);
+    timer.setText("mm:ss");
+    timer.setLocalTranslation(2.5f, 5, 3.9f);
+    timer.setLocalScale(0.1f);
+    timer.setLocalRotation(new Quaternion().fromAngleAxis(180 * FastMath.DEG_TO_RAD, new Vector3f(0, 1, 0)));
```
Comment (FlorisXll) [inline src/main/java/nl/tudelft/pixelperfect/Game.java]: Checkstyle warning: line is too long.

**Comment by FlorisXll** [comment]
I would recommend to activate Checkstyle locally so you can fix all warnings at once.

**Comment by FlorisXll** [comment]
Furthermore LGTM.

PS: I like the blue dashboard

File: src/main/java/nl/tudelft/pixelperfect/Game.java (hunk 1 above)
Comment (wouterz) [inline src/main/java/nl/tudelft/pixelperfect/Game.java]: Swapped them, but Im not getting warnings about this specific thing, are you also using the rules of checkstyle.xml located in the project? David also seems to have other checkstyle warnings than I do in another PR so just wondering

**Comment by wouterz** [comment]
Fixed all checkstyle errors except a line that was too long (109, max 100), this should still easily fit on all modern screens so leaving that as is.
